docs/src/communication/overview.md

```diff
@@ -0,0 +1,4 @@
+# Communication
+
+- Use GRPC for communication between front and daemon
+- Use GRPC for communication between daemon and datastore
```

Comment:
I've added this file for adjusting mdbook structure restriction.
Please update the content if you need.